tests/test_mapping/test_mapping_helpers.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import pandas as pd
+import numpy as np
+
+# Local Imports
+from src.mapping.mapping_helpers import update_ref_list
+
+
+class TestUpdateRefList(object):
+    """Tests for update_ref_list."""
+
+    @pytest.fixture(scope="function")
+    def full_input_df(self):
+        """Main input data for update_ref_list tests."""
+        columns = ["reference", "instance", "formtype", "cellnumber"]
+        data = [
+            [49900001031, 0.0, 6, 674],
+            [49900001530, 0.0, 6, 805],
+            [49900001601, 0.0, 1, 117],
+            [49900001601, 1.0, 1, 117],
+            [49900003099, 0.0, 6, 41],
```

Comment:
it's perhaps nicer to have an integer for instance. There should be some examples of tests where we've got round the problem of it creating a different type to the one we expect, for example using .astype or ignoring the types when we do the assert.